flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js

```diff
@@ -0,0 +1,240 @@
+function airplaneCanBoard() {
+    const busDC2 = SimVar.GetSimVarValue("L:A32NX_ELEC_DC_2_BUS_IS_POWERED", "Bool");
+    const busDCHot1 = SimVar.GetSimVarValue("L:A32NX_ELEC_DC_HOT_1_BUS_IS_POWERED", "Bool");
+    const gs = SimVar.GetSimVarValue("GPS GROUND SPEED", "knots");
+    const isOnGround = SimVar.GetSimVarValue("SIM ON GROUND", "Bool");
+    const eng1Running = SimVar.GetSimVarValue("ENG COMBUSTION:1", "Bool");
+    const eng2Running = SimVar.GetSimVarValue("ENG COMBUSTION:2", "Bool");
+
+    if (gs > 0.1 || eng1Running || eng2Running || !isOnGround || (!busDC2 && !busDCHot1)) {
+        return false;
+    }
+    return true;
```

Comment:
I do not want to be annoying, but why not just return the evaluation?  :see_no_evil:

```suggestion
return !(gs > 0.1 || eng1Running || eng2Running || !isOnGround || (!busDC2 && !busDCHot1));
```

Reply:
Can do that, and it's not annoying dw 😄